pkg/sql/exec_log.go

```diff
@@ -323,6 +323,14 @@ func (p *planner) maybeLogStatementInternal(
 
 			skippedQueries := telemetryMetrics.resetSkippedQueryCount()
 
+			var sqlInstanceIDs []string
+			if len(queryLevelStats.SqlInstanceIds) > 0 {
+				sqlInstanceIDs = make([]string, 0, len(queryLevelStats.SqlInstanceIds))
+				for sqlId := range queryLevelStats.SqlInstanceIds {
+					sqlInstanceIDs = append(sqlInstanceIDs, sqlId.String())
+				}
+			}
```

Comment:
May I recommend you make the event type contain an array of integers (instead of an array of strings), then include the `queryLevelStats.SqlInstanceIds` slice directly into the event without conversion to string.